src/app/components/BCP47Tag.svelte

```diff
@@ -14,8 +14,19 @@
   }
 </script>
 
+<style>
+  :global().input-field__label-bcp {
+    font: var(--main-font);
+    font-size: var(--s) !important;
+    font-weight: normal !important;
```

Comment:
pwease do not resort to `!important`. `!important` is a measure of last resort, because it actually overrides the user settings, and the users' settings are what gives us the correct font-size. Why did you use `!important` here, and can that be solved with a more specific selector?

Reply:
correct if I am wrong, Svelte first compile the page and then the components within the page. Since input field on the landing page have different design, I would have to use important to overwrite the input field style. I personally don't think this is a good solution, I will try a different implementation

Reply:
I totally agree on you and `!important` is definitely not a best practice. Rather than just overriding the style, I made some changes to introduce the size attribute for the input component.